test/steps/data.go

```diff
@@ -62,6 +69,14 @@ func getNamespaceName() string {
 	return framework.GenerateNamespaceName("cucumber")
 }
 
+func createTemporaryFolder() string {
```

Comment:
wouldn't be better to have that one in `util.go` from framework ?

Reply:
ok